ui/webui/test/check-basic

```diff
@@ -94,7 +94,7 @@ class TestBasic(anacondalib.VirtInstallMachineCase):
         i.open(step="installation-method")
 
         # Back button should be disabled on the first screen
-        b.wait_visible(f"#installation-back-btn:not([aria-disabled={True}]")
+        b.wait_visible("#installation-back-btn:not([aria-disabled=false]")
```

Comment:
This does not allign now with the comment in the line above.

Reply:
I was just confused by the double negatio. 

Can you please iron it out a bit:

b.wait_visible(f"#installation-back-btn[aria-disabled=true]")